### DISCUSSION TITLE
Better customization of main character.

### DISCUSSION THREAD

**Discussion by AircoDHK** [endless-sky/endless-sky#8658] [Ideas]
**SPOILERS AHEAD !**

In this thread I will not be talking much about the customization of your character such as gender or things like that which i think is not urgent because they don't change much of the gameplay.

I'll rather talk about the choice you could make of starting your life as an Hai, an Arach, and basically any alien factions with maybe the exception of Draks, Pugs and Quargs. This choice could bring important changes to the different storylines already available and pave the way for new unique missions based on your race. I thought of this when I first went into coalition space and saw all these civilian ships that i will never use because I already have better ships. This way, if you allow the player to choose his race you could also have a different difficulty rating. (Coalition space would be easy: no pirate at all and pretty good ships overall compared to humans, Humans would be normal difficulty: mid ships, a moderate pirate threat, and gegno would be Hard: ships with no shield and overall inefficient outfits with a civil war.). This is also interesting because I would love to take part in gagno's civil war from the inside. this could be the equivalent to human's Free world campaign. Finally it would enable the player to truly appreciate every ship that is present in the game which is hard work from every developper on here and i would love to enjoy to the fullest their work.

**Comment by Zitchas** [comment]
This is definitely an idea that we are working towards. Having multiple places to start from would f definitely add to the replayability.

**Comment by mOctave** [comment]
A couple things you might want to look into:
- There is already a Hai start somewhere (which requires some of the main plot to be completed before being unlocked), although I can't remember off the top of my head whether it's in game yet. As Zitchas said, there will probably be more in the future.
- Zoura / AvianGeneticist has a plugin [Play as Sestor](https://github.com/AvianGeneticist/Play-As-Sestor) which adds another start.

P.S. You don't need to mark spoilers on GitHub, it's presumed that anyone looking at this either knows everything or isn't afraid of spoilers. That said, feel free to mark them if you want to.
